doc/release-notes.md

```diff
@@ -53,6 +53,27 @@ the Linux kernel, macOS 10.8+, and Windows Vista and later. Windows XP is not su
 Bitcoin Core should also work on most other Unix-like systems but is not
 frequently tested on them.
 
+Notes for 0.15.0
+================
+
+Current SegWit Support
+----------------------
+
+Version 0.15.0 supports adding a segregated witness address via the `addwitnessaddress` RPC, but
+please note that this is a testing/expert RPC, which does not guarantee recovery from backup. Only use
+this RPC if you know what you are doing. More complete wallet support for segregated witness is coming
+in a next version.
```

Comment:
Is this going to be in a point release, or 0.16.0 ? I think we should mention which here, because one could be a month away, while the other is 6 months or more. If not, instead of "is coming in a next version.", maybe just "will be available shortly." ?

Reply:
Segwit wallet support is aimed for 0.15.1 I believe?

Reply:
meshcollider is correct.